recipes/openexr/2.3.0/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+set(CONAN_BASIC_SETUP_ARGS)
+if(APPLE)
```

Comment:
did you try https://docs.conan.io/en/latest/reference/build_helpers/cmake.html#patch-config-paths-experimental instead ?

Reply:
Or https://docs.conan.io/en/latest/howtos/manage_shared_libraries/rpaths.html#different-approaches

Reply:
I think that fixes another problem.
openexr generates some sources using in-tree built executables that depends on (shared) libraries.
On Apple these exes do not work when the dylibs do not contain rpaths.
I can fix this by using rpaths for the dylibs in the build tree, but remove them when installing:
```
    list(APPEND CONAN_BASIC_SETUP_ARGS KEEP_RPATHS)
    set(CMAKE_SKIP_INSTALL_RPATH 1)
```

Reply:
When using `CMAKE_BUILD_WITH_INSTALL_RPATH` and `CMAKE_INSTALL_RPATH `, don't we want an empty rpath?

Reply:
I see thanks for clarifying !